api-server/src/events/events.authenticated.controller.ts

```diff
@@ -40,16 +41,25 @@ import {
   PaginationDto,
 } from './dto/pagination-dto';
 import { validateAndExtractOptionalDateTimeFilters } from './utils/validateAndExtractOptionalDatetimeFilters';
+import { AuthenticatedUserGuard } from '../authentication/auth-guards/authenticated-user.guard';
+import { PartnerAdminGuard } from '../authentication/auth-guards/partner-admin.guard';
+import UsersService from '../users/users.service';
+import { RequestWithUserInfo } from '../users/dto/RequestWithUserInfo';
+import { Op } from 'sequelize';
 
 @Controller(`${VERSION_1_URI}/authenticated/events`)
-@UseGuards(AuthGuard)
+@UseGuards(AuthenticatedUserGuard)
```

Comment:
In retrospect I think I could have just used the PartnerGuard on all of these and then applied the additional admin guard on the ones that needed, still debating in my head what is more readable, effect would be the same :shrug: